src/Control/Lens/Internal/Context.hs

```diff
@@ -63,9 +61,8 @@ class IndexedFunctor w where
 -- "Parameterized Notions of Computation" <http://bentnib.org/paramnotions-jfp.pdf>
 -- and that construction is dualized here.
 class IndexedFunctor w => IndexedComonad w where
-#if __GLASGOW_HASKELL__ >= 708
```

Comment:
Now that this code is gone, we should be able to get rid of the `{-# LANGUAGE CPP #-}` line above.